Makefile

```diff
@@ -329,3 +367,10 @@ src/static.c: $(BUILD) Makefile $(APP_MK) $(MOD_MK)
 	done
 	@echo "  NULL"  >> $@
 	@echo "};"  >> $@
+
+.PHONY: info2
+info2:
```

Comment:
You can now replace:
```
.PHONY: info2
info2:
```
by:

```
info::
```

https://www.gnu.org/software/make/manual/html_node/Double_002dColon.html#Double_002dColon

https://github.com/baresip/re/commit/432fa6a92f1b369e4e54ae0841c4808b686af025